paper-shadow.html

```diff
@@ -23,34 +23,90 @@
 -->
 
 <link href="../polymer/polymer.html" rel="import">
+<link href="../layout/layout.html" rel="import">
+<dom-module id="paper-shadow">
+  <style>
+    paper-shadow {
+      display: block;
+      position: relative;
+      pointer-events: none;
+    }
+
+    paper-shadow #shadow-bottom,
+    paper-shadow #shadow-top {
+      border-radius: inherit;
+    }
+
+    paper-shadow #shadow-bottom[animated],
+    paper-shadow #shadow-top[animated] {
+      transition: box-shadow 0.28s cubic-bezier(0.4, 0, 0.2, 1);
+    }
+
+    paper-shadow #shadow-top.paper-shadow-z-1 {
+      box-shadow: none;
+    }
+
+    paper-shadow #shadow-bottom.paper-shadow-z-1 {
+      box-shadow: 0 1px 4px 0 rgba(0, 0, 0, 0.37);
+    }
```

Comment:
@morethanreal Please consider #33 as an adaptation to be merged into this branch per your feedback.